mne/viz/ica.py

```diff
@@ -277,6 +278,11 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
         defaults to [7., 6.].
     show : bool
         Show figure if True.
+    reject : 'auto' | dict | None
+        Allows to use a different rejection parameter to drop epochs. If None, 
+        it does not apply a rejection. If 'auto', it applies the rejection fitted
+        with the ICA object if it exists, else, it does not apply any
```

Comment:
I don't understand that part: `else, it does not apply any defaults to 'auto'`. What do you mean by "appling defaults to 'auto'"? BTW: does the `else` refer here to using `dict` or using something else (neither a dict, `'auto'` nor `None`)?

Reply:
Indeed that is misleading, the 'else' applies to the existence of a reject param used when fitting the ICA, I will rewrite this part.